exercises/error-handling/README.md

```diff
@@ -0,0 +1,24 @@
+# Error Handling
```

Comment:
Has this been generated by `configlet`? I'd expect the metadata to show up

Reply:
Yes, I generated it with ```configlet```.

Reply:
Perfect! On a closer look, the normal metadata I would expect is not actually in the `problem-specifications` yet!